product_medical/migrations/18.0.1.0.0/pre-migrate.py

```diff
@@ -0,0 +1,42 @@
+#  Copyright 2024 Simone Rubino - Aion Tech
+#  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+MODEL_TO_RENAMED_FIELDS = {
+    "product.template": [
+        ("ppe_category_id", "medical_ppe_category_id"),
+        ("in_vitro_diagnostic", "medical_in_vitro_diagnostic"),
+        ("in_vitro_diagnostic", "medical_in_vitro_diagnostic"),
+        ("conformity_declaration_ids", "medical_conformity_declaration_ids"),
+        ("doc_lot_related", "medical_doc_lot_related"),
+        ("doc_validity_date", "medical_doc_validity_date"),
+        (
+            "ce_certificate_medical_class_ids",
+            "medical_ce_certificate_medical_class_ids",
+        ),
+        ("ce_certificate_validity_date", "medical_ce_certificate_validity_date"),
+        ("notified_body_id", "medical_notified_body_id"),
+    ]
+}
+
+
+def _rename_fields(env):
+    openupgrade.rename_fields(
+        env,
+        [
+            (
+                model_name,
+                model_name.replace(".", "_"),
```

Comment:
To get the name of the table, that would be better to use `env[model_name]._table`, but at the same time:
- using the registry in a pre script is sometimes not a good idea
- no issue here for `product.template` data model, that will work

So not blocking :man_shrugging: